### ISSUE TITLE
mipmap support

### ISSUE THREAD

**Issue by danielgomezrico** [google/material-design-icons#97]

**Comment by jestelle** [comment]
Can you clarify what exactly you're looking for here?

**Comment by danielgomezrico** [comment]
When you create the drawables it create the icons in the `drawable-x`, `drawable-h`,... folders but I´m using `mipmap-x`, `mipmap-h`, ... folders for my images because, as I understood, it is the new way of images management for android, `drawables` folder will be used for shapes and those things and `mipmap` folders for images.

I´m not sure if that is true... but if yes maybe a spinner to select if you want the images into drawables folders or into mipmap folders.

**Comment by jestelle** [comment]
I think you may be misunderstanding the value of using mipmap densities.

To the best of my knowledge, mipmap density folders are only used for one reason, and that is a way of keeping *all* density versions of an asset included in your app build, even when you're doing density specific builds.

The main reason people often want to do this is for their launcher icon, because some launchers on Android will pull from greater density buckets than normal, and by using mipmap those assets are always available.

This might be a little confusing – maybe this article explains it more clearly for you:
https://androidbycode.wordpress.com/2015/02/14/goodbye-launcher-drawables-hello-mipmaps/

In short, I don't believe there is any value in us providing these in mipmap folders.

**Comment by danielgomezrico** [comment]
you are right! opa, thanks!!!!!! :hand:
